databases/metadata/src/main/functions/04_ontology_hierarchy.sql

```diff
@@ -0,0 +1,32 @@
+SET search_path = public, pg_catalog;
+
+--
+-- A function that returns all of the ontology subclasses of the class with the given
+-- ontology_version and IRI.
+--
+CREATE OR REPLACE FUNCTION ontology_class_hierarchy(VARCHAR, VARCHAR)
```

Comment:
As expected, looks like most of the work is in the recursive loop. But it's also quite fast, since everything's indexed by ontology version, so looks good :+1: 

Thanks for humoring me with this.